community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-pipelines.ipynb

```diff
@@ -165,7 +165,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        # base_image="gcr.io/google.com/cloudsdktool/cloud-sdk:latest",
</pre><p>Is this commented out line supposed to be here (I've seen it in a couple components)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>I have removed the commented base image.</p>